test/Generators/Microsoft.Gen.Logging/Unit/ParserTests.cs

```diff
@@ -130,20 +130,6 @@ partial class C
                 }");
     }
 
-    [Fact]
-    public async Task InvalidMethodName()
-    {
-        const string Source = @"
-                partial class C
-                {
-                    [LogMethod(0, LogLevel.Debug, ""M1"")]
-                    static partial void /*0+*/__M1/*-0*/(ILogger logger);
-                }
-            ";
-
-        await RunGenerator(Source, DiagDescriptors.InvalidLoggingMethodName);
```

Comment:
We can still have this test to ensure that no diagnostics are reported

Reply:
DOne.